services/f-globalisation/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@justeat/f-globalisation",
+  "description": "Fozzie Globalisation – A utility which wires up vue-i18n within your Smart Component",
+  "version": "0.1.0",
+  "main": "dist/f-globalisation.umd.min.js",
+  "files": [
+    "dist"
+  ],
+  "homepage": "https://github.com/justeat/fozzie-components/tree/master/packages/f-globalisation",
+  "contributors": [
+    "Github contributors <https://github.com/justeat/fozzie-components/graphs/contributors>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:justeat/fozzie-components.git"
+  },
+  "bugs": {
+    "url": "https://github.com/justeat/fozzie-components/issues"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "keywords": [
+    "fozzie"
+  ],
+  "scripts": {
+    "prepublishOnly": "yarn lint && yarn build",
```

```suggestion
    "prepublishOnly": "yarn lint && yarn test && yarn build",
```
We should add test back into this; because we removed it earlier due to an issue in the generator